packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -581,20 +581,22 @@ export class Repository extends Construct implements IRepository {
     if (props.fileSystem instanceof MountableEfs && !props.fileSystem.accessPoint) {
```

Comment:
I noticed that we also don't document the admin credentials secret being deployed in the "Resources Deployed" section of the `Repository` construct's TSDoc header. Do you mind adding that as part of this PR?